samples/AvatarWithRoundedCorner/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿
+
+namespace AvatarWithRoundedCorner
+{
+    using System;
+    using System.Numerics;
+    using ImageSharp;
+    using SixLabors.Shapes;
+
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
I disagree (obviously 😉 ) My logic for having lot of individual projects is for the fact they are supposed to be self contained demonstrations on how a user could implement such a thing... if they are all bundled into a larger single project then 1 its harder to see what the dependencies are, and 2 it becomes tempting to share code across the samples which, in my opinion, shouldn't happen so as to make it simpler to follow (preferably without even having to download from github) but If people feel strongly enough I'll to change it.